region/maridia/inner-green/Shaktool Room.json

```diff
@@ -230,20 +329,110 @@
     },
     {
       "link": [2, 1],
-      "name": "Carry Grapple Teleport (Bottom Position)",
+      "name": "Carry Grapple Teleport (Top Position, Symmetric)",
+      "entranceCondition": {
+        "comeInWithGrappleTeleport": {
+          "blockPositions": [[3, 12]]
+        }
+      },
+      "requires": [
+        "h_ShaktoolSymmetricFlag"
+      ],
+      "bypassesDoorShell": true,
+      "exitCondition": {
+        "leaveWithGrappleTeleport": {
+          "blockPositions": [[3, 12]]
+        }
+      },
+      "setsFlags": ["f_ShaktoolDoneDigging"]
+    },
+    {
+      "link": [2, 1],
+      "name": "Carry Grapple Teleport (Bottom Position, Vanilla)",
       "entranceCondition": {
         "comeInWithGrappleTeleport": {
           "blockPositions": [[3, 13]]
         }
       },
-      "requires": [],
+      "requires": [
+        "h_ShaktoolVanillaFlag"
+      ],
       "bypassesDoorShell": true,
       "exitCondition": {
         "leaveWithGrappleTeleport": {
           "blockPositions": [[3, 13]]
         }
       }
     },
+    {
+      "link": [2, 1],
+      "name": "Carry Grapple Teleport (Bottom Position, Symmetric)",
+      "entranceCondition": {
+        "comeInWithGrappleTeleport": {
+          "blockPositions": [[3, 13]]
+        }
+      },
+      "requires": [
+        "h_ShaktoolSymmetricFlag"
+      ],
+      "bypassesDoorShell": true,
+      "exitCondition": {
+        "leaveWithGrappleTeleport": {
+          "blockPositions": [[3, 13]]
+        }
+      },
+      "setsFlags": ["f_ShaktoolDoneDigging"]
+    },
+    {
+      "link": [2, 2],
+      "name": "Come In With Shaktool Done Digging",
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        "f_ShaktoolDoneDigging"
+      ],
+      "clearsObstacles": ["A", "B"]
+    },
+    {
+      "link": [2, 2],
+      "name": "Shaktool Room Bounce the Snails Around the Room",
```

Comment:
I think 
Expert: Gravity or yard hit
Very Hard: Morph and Gravity
Extreme: patience? 

Its pretty easy to miss a shot then just walk into the snail for damage.
Or get slowed by the water when you don't want to be.

Reply:
After playing around with this some more, I'm thinking Morph may not be important to model here. Gravity is the item that makes the big difference in difficulty. With Gravity, you have a lot more freedom in how you can kick the snails around, and with that there doesn't seem to be much risk of getting them both stuck off-camera; having a second snail available to use as a spare helps a lot.

Without Gravity, it's fairly different, as it needs more of a plan. I'm thinking it maybe makes sense to split off the suitless version as a separate notable strat which can be in Expert. And then we don't have to introduce a new tech ("canTrickyDodgeEnemies"). Snail hit seems reasonable to assume for the Expert suitless version, with damageless suitless being assumed in Extreme, whether by patience or by carefully timed movement.